[pull] primary from flybywiresim:primary (#108)
Valastiri/fbw-userguide: docs/beginner-guide/after-landing.md

    The level of detail in this guide is meant to get a FlyByWire A320neo beginner from the runway to the designated destination gate.

    A *beginner* is defined as someone familiar with flying a GA aircraft or different types of airliners. Aviation terminology and know-how is a requirement to fly any airliner even in Microsoft Flight Simulator.

    Further reading: [A320 Autoflight](https://www.smartcockpit.com/aircraft-ressources/A319-320-321-Autoflight.html)<br/>
    Also you will find many great videos on YouTube on how to fly the FlyByWire A32NX.<br/>
    Check out the FlyByWire YouTube Channel as well: [FlyByWire on YouTube](https://www.youtube.com/c/FlyByWireSimulations/playlists)

---

## Pre-requisites

Aircraft has vacated the runway completely and has been brought to a stop on the taxiway as per previous chapters.

![Runway vacated](../assets/beginner-guide/after-landing/Runway_vacated.png "Runway vacated")

Download the [FlyByWire Checklist](../assets/FBW_A32NX_CHECKLIST.pdf) here.

## Chapters / Phases

This guide will cover these phases:

1. [After Landing](#1-after-landing)
2. [Taxi to Gate](#2-taxi-to-gate)

---

### 1. After Landing

**Situation:**

> - Aircraft has vacated the runways completely and has come to a  stop on the taxi way as per previous chapters.

Reviewer: ```suggestion
- Aircraft has vacated the runways completely and has come to a stop on the taxi way as per previous chapters.
```

- Flaps and Ground Spoilers are still deployed.
- ATC has been informed that we vacated the runway.

!!!info "Simulation vs. Real Life"

>     In real live the A320 will have two pilots which can actually do things in parallel. Talking to ATC, taxing the aircraft and do the after landing tasks.

Reviewer: ```suggestion
    In real life the A320 will have two pilots who can actually do things in parallel. Talking to ATC, taxing the aircraft and do the after landing tasks.
```

    ![APU Start](../assets/beginner-guide/after-landing/APU-Start.png "APU Start"){width=120}

- Turn `ANTI ICE` OFF if not required.

- Set your exterior lights:
    - Set your `NOSE` light to TAXI.
    - Set `RWY TURN OFF` light to ON.
    - Set your `STROBE` lights to AUTO/OFF (If you are crossing runway keep them ON until you vacate the runway).
    - For `LAND` lights, you can choose to retract them or turn them off.

        ![After landing lights](../assets/beginner-guide/after-landing/Taxi-light-on.png "After landing lights"){width=320}

- Turn the brake fan (`BRK FAN`) on if you get a hot brakes ECAM warning.

    ![Hot Brakes Brake Fan on](../assets/beginner-guide/after-landing/Brake-Fan.png "Hot Brakes Brake Fan on"){width=640}

- Complete the **After Landing** checklist.
    ```
    FLAPS: RETRACTED
    SPOILERS: DISARMED
    APU: START
    RADAR: OFF
    PREDICTIVE WINDSHEAR SYSTEM: OFF        
    ```

This concludes *After Landing*

### 2. Taxi to Gate

**Situation:**

- Aircraft on taxiway directly after runway.
- **After Landing** checklist is completed.
- ATC Ground has given taxi instructions.

**Taxiing**

> Make sure to also read the Taxi section of the [Engine Start and Taxi](/engine-start-taxi/#taxi) chapter.

Reviewer: ```suggestion
Make sure to also read the Taxi section of the [Engine Start and Taxi](engine-start-taxi.md/#taxi) chapter.
```
I'm pretty sure this doesn't work and should be as such.
